sdk/iot/core/tests/cmocka/az_iot_hub_client_tests.c

```diff
@@ -0,0 +1,20 @@
+#include <az_iot_hub_client.h>
```

Comment:
@danewalton - this is missing the (c) header - please add it in a new review.

Reply:
ah thank you

Reply:
will add in the next review for other API's